src/main/resources/reference.conf

```diff
@@ -96,3 +97,7 @@ autoFreeze {
   autoFreezeAfter = 30 minutes
   autoFreezeCheckScheduler = 1 minute
 }
+
+jupyterContentSecurityPolicy {
+  contentSecurityPolicy = "frame-ancestors 'self' http://localhost:3000 https://bvdp-saturn-prod.appspot.com https://bvdp-saturn-dev.appspot.com; script-src 'self' 'unsafe-inline'  https://apis.google.com ; style-src 'self' 'unsafe-inline'; connect-src 'self' wss://*.broadinstitute.org:* *.googleapis.com"
```

Comment:
Can you add `https://localhost:443`? I think I had added that on one of the single-cell-portal clusters. We should also find out what the prod SCP hosts are, but we can always modify this config later.

Reply:
They are using Leo in an iframe?